math/check_factorial.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file
+ * @brief A simple program to check if the given number is a factorial of some
+ * number or not.
+ * @author [Divyajyoti Ukirde](https://github.com/divyajyotiuk)
+ */
+#include <cassert>
+#include <iostream>
+
+/**
+ * Function to check if the given number is factorial of some number or not.
+ * @param n number to be checked.
+ * @return if number is a factorial, returns true, else false.
+ */
+
+bool is_factorial(int n) {
```

Comment:
wouldn't it be better to use the fixed number type as below?
```suggestion
bool is_factorial(uint64_t n) {
```
similarly, the variable `i` on line 20 could probably be `uint32_t`?

Reply:
Yes, will make the changes. Will help dealing with large numbers.